packages/flutter/lib/src/material/divider.dart

```diff
@@ -111,3 +114,109 @@ class Divider extends StatelessWidget {
+///
+/// Dividers can be used in lists, [Drawer]s, and elsewhere to separate content
+/// horizontally. To create a one-pixel divider between items in a list,
+/// consider using [ListTile.divideTiles], which is optimized for this case.
+///
+/// The box's total width is controlled by [width]. The appropriate
+/// padding is automatically computed from the width.
+///
+/// See also:
+///
+///  * [PopupMenuDivider], which is the equivalent but for popup menus.
+///  * [ListTile.divideTiles], another approach to dividing widgets in a list.
+///  * <https://material.google.com/components/dividers.html>
+class VerticalDivider extends StatelessWidget {
+  /// Creates a material design divider.
+  ///
+  /// The width must be positive.
+  const VerticalDivider({
+    Key key,
+    this.width = 16.0,
+    this.indent = 0.0,
+    this.color
+  }) : assert(width >= 0.0),
+       super(key: key);
+
+  /// The divider's width.
+  ///
+  /// The divider itself is always drawn as one device pixel thick
+  /// line that is centered within the width specified by this value.
+  ///
+  /// A divider with a [width] of 0.0 is always drawn as a line with a width
+  /// of exactly one device pixel, without any padding around it.
+  final double width;
+
+  /// The amount of empty space to the left of the divider.
+  final double indent;
+
+  /// The color to use when painting the line.
+  ///
+  /// Defaults to the current theme's divider color, given by
+  /// [ThemeData.dividerColor].
+  ///
+  /// ## Sample code
+  ///
+  /// ```dart
+  /// Divider(
+  ///   color: Colors.deepOrange,
+  /// )
+  /// ```
+  final Color color;
+
+  /// Computes the [BorderSide] that represents a divider of the specified
+  /// color, or, if there is no specified color, of the default
+  /// [ThemeData.dividerColor] specified in the ambient [Theme].
+  ///
+  /// The `width` argument can be used to override the default width of the
+  /// divider border, which is usually 0.0 (a hairline border).
+  ///
+  /// ## Sample code
+  ///
+  /// This example uses this method to create a box that has a divider above and
+  /// below it. This is sometimes useful with lists, for instance, to separate a
+  /// scrollable section from the rest of the interface.
+  ///
+  /// ```dart
+  /// DecoratedBox(
+  ///   decoration: BoxDecoration(
+  ///     border: Border(
+  ///       top: Divider.createBorderSide(context),
+  ///       bottom: Divider.createBorderSide(context),
+  ///     ),
+  ///   ),
+  ///   // child: ...
+  /// )
+  /// ```
+  static BorderSide createBorderSide(BuildContext context, { Color color, double width = 0.0 }) {
+    assert(width != null);
+    return BorderSide(
+      color: color ?? Theme.of(context).dividerColor,
+      width: width,
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: width,
+      child: Center(
+        child: Container(
+          height: 0.0,
```

Comment:
Hey, guys! 👋 Shouldn't this line (`line #211`) be `width: 0.0,` instead of `height: 0.0,`? 😋 I just tried using `VerticalDivider()`, but since the `height` of the container is set to `0.0`, the divider/border is not rendered.

Reply:
Does that fix the problem for you? But that does seem like what it should be!

Reply:
Hey, @jslavitz!

So, the following code works for me only if I change that line from `height: 0.0,` to `width: 0.0,`.

```dart
Container(
  height: 24.0,
  child: Row(
    children: <Widget>[
      Text('Hey.'),
      VerticalDivider(),
    ],
  ),
),
```

Does this work for you? 😅